tests/api/api/test_functions.py

```diff
@@ -228,12 +231,31 @@ async def test_list_functions_with_hash_key_versioned(
     assert list_functions_results[0]["metadata"]["hash"] == hash_key
 
 
-@pytest.mark.parametrize("post_schedule", [True, False])
-def test_delete_function_with_schedule(
+@pytest.mark.parametrize(
+    "post_schedule, kind",
+    [
+        (True, "job"),
+        (False, "job"),
+        (False, "remote"),
+    ],
+)
+@pytest.mark.parametrize("use_v2_deletion_endpoint", [True, False])
```

Comment:
```suggestion
@pytest.mark.parametrize("endpoint_prefix, expected_status", 
        [
            ("v1", HTTPStatus.NO_CONTENT.value),
            ("v2", HTTPStatus.ACCEPTED.value)
        ]
)
```

Reply:
@TomerShor default value is with /v1/ (it is configured in fastapi.base_url), I'm not sure if there is a sense to do all logic with `v1` explicitly since it's already there